server/routes/taggedBailRoutes.ts

```diff
@@ -103,6 +105,34 @@ export default class TaggedBailRoutes {
     })
   }
 
+  public remove: RequestHandler = async (req, res): Promise<void> => {
+    const { caseloads, token } = res.locals.user
+    const { nomsId, id } = req.params
+    const prisonerDetail = await this.prisonerService.getPrisonerDetail(nomsId, caseloads, token)
+    const adjustment = await this.adjustmentsService.get(id, token)
+    if (!adjustment) {
+      return res.redirect(`/${nomsId}`)
+    }
+
+    const adjustmentType = adjustmentTypes.find(it => it.url === 'tagged-bail')
+    if (!adjustmentType) {
+      return res.redirect(`/${nomsId}`)
+    }
+
+    const sentencesAndOffences = await this.prisonerService.getSentencesAndOffences(prisonerDetail.bookingId, token)
+    const sentencesByCaseSequence = getActiveSentencesByCaseSequence(sentencesAndOffences)
+    const sentencesForCaseSequence = sentencesByCaseSequence.find(
+      it => it.caseSequence === adjustment.taggedBail.caseSequence,
+    )
+    const sentenceAndOffence = sentencesForCaseSequence.sentences.sort(
```

Comment:
I think this sort function could be refactored somewhere. It appears a lot through all the tagged bail journeys. (Similar to getActiveSentencesByCaseSequence)

Reply:
I've created the method `getMostRecentSentenceAndOffence(sentencesAndOffences: PrisonApiOffenderSentenceAndOffences[])` in `utils.ts` and have replaced all usages of the sort with the call to this new utility method 👍